vslib/SwitchConfig.cpp

```diff
@@ -79,6 +79,15 @@ bool SwitchConfig::parseSwitchType(
     {
         switchType = SAI_VS_SWITCH_TYPE_NVDA_MBF2H536C;
     }
+    else if (st == SAI_VALUE_VS_SWITCH_TYPE_NPU)
```

Comment:
nope, this should not be that way, see that NPU seems like generic, look at other implementation that they have specific names, since SAI_VS_SWITCH_TYPE_NVDA_MBF2H536C is already defined, then you you should use SAI_VALUE_VS_SWITCH_TYPE_NVDA_MBF2H536C to se its value

Reply:
@kcudnik, The problem we are faced with is that support for NVDA_MBF2H536C is not yet available in public sonic-buildimage repo. Those are still in the nvidia-sonic buildimage repo. Nvidia is planning to eventually upstream those, but until then we would like to have a way to run vstests without depending on NVDA_MBF2H536C hwsku. Guohan had suggested to create a "generic" NPU hwksu that could be used for overcoming this issue. Please see https://github.com/sonic-net/sonic-buildimage/pull/15604 and https://github.com/sonic-net/sonic-buildimage/pull/15695 which are related sonic-buildimage PRs.

Reply:
this should be SAI_VALUE_VS_SWITCH_TYPE_NPU_2P make it specific. then, we can create such vslib, just copy paste NVDA_MBF2H536C, change to NPU_2P. if we need more specific name, we can say NPU_SIMU_2P, meaning this is to 2 port npu simulator.

Reply:
thought about it, maybe we can just rename to SAI_VALUE_VS_SWITCH_TYPE_NPU_SIMU_2P and it is kind of alias to nvda_mbf2h536c

Reply:
Okay, I have changed the name to SAI_VALUE_VS_SWITCH_TYPE_NPU_SIMU_2P.